docs/standards/generic-standards/lsp23-linked-contracts-factory.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_label: 'LSP23 - Linked Contracts Factory'
+sidebar_position: 7
+---
+
+# LSP23 Linked Contracts Factory
+
+:::info Standard Document
+
+[LSP23 - Linked Contracts Factory](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-23-LinkedContractsFactory.md)
+
+:::
+
+## Introduction
+
+Deploying smart contracts that are interdependent can be a complex process. Often, two or more contracts need each other's addresses at deployment, creating a circular dependency that can be challenging to resolve.
+
+![Circular Depedency](/img/standards/lsp23/circular-dependency.jpeg)
+
+The LSP23 Linked Contracts Factory addresses this issue by providing a unified interface for deploying linked contracts with an optional post-deployment execution. This standard simplifies the deployment of complex contract systems, making it easier for developers to deploy interdependent contracts without the need for manual intervention.
+
+## What does this standard represent?
+
+This standard represents a smart contract factory that allows users to deploy primary and secondary contracts that are linked together. It also supports deploying contracts as `ERC1167` proxies. The factory can execute post-deployment modules to perform additional logic after deploying the contracts (for example, it could pass the address of the secondary to the primary contract).
+
+![LSP23 Flow](/img/standards/lsp23/lsp23-flow.jpeg)
+
+As [`LSP16`](./lsp16-universal-factory.md), the `LSP23` standard addresses the issue of deterministic address generation for contracts across different chains by generating the salt for contract deployment within the function, ensuring a fully decentralized deployment process. This approach guarantees consistent contract addresses across multiple chains and prevents address squatting, enhancing the security and integrity of the deployment process.
+
+## Specification
+
+<!--
+Will uncomment when the dodocs has generated the docs for the LSP23 contract.
```

Comment:
I suggest to add the auto-generated ABI docs already, even if they are not perfect.
And we can update them afterwards, once @b00ste has reviewed the Natspecs of LSP23.